src/ext/java/org/opentripplanner/ext/siri/TimetableHelper.java

```diff
@@ -153,7 +153,7 @@ public static TripTimes createUpdatedTripTimes(final Graph graph, Timetable time
                     }
 
                     //Flag as recorded
-                    newTimes.setRecorded(callCounter, true);
+                    newTimes.setRecorded(callCounter, true); // TODO only set if actual arrival time is not null
```

Comment:
Try to avoid the "old style" of adding comments at the end of a line, put it on the line above. The reason is that if the code is auto-formated the comment might end up on the line below - then it might be very confusing, because it suddenly apply to the next line of code.